docs_2.0/manual_EN/5.deployment-and-installation/1.resource-preparations.md

```diff
@@ -0,0 +1,181 @@
+| m4               | Any mainstream version | -                                                         |
+| git              | Any mainstream version | -                                                         |
+| wget             | Any mainstream version | -                                                         |
+| unzip            | Any mainstream version | -                                                         |
+| xz               | Any mainstream version | -                                                         |
+| readline-devel   | Any mainstream version | -                                                         |
+| ncurses-devel    | Any mainstream version | -                                                         |
+| zlid-devel       | Any mainstream version | -                                                         |
+| gcc              | 7.1.0 or above         | You can run `gcc -v` to check the gcc version.            |
+| gcc-c++          | Any mainstream version | -                                                         |
+| cmake            | 3.5.0 or above         | You can run `cmake --version` to check the cmake version. |
+| gettext          | Any mainstream version | -                                                         |
+| curl             | Any mainstream version | -                                                         |
+| redhat-lsb-core  | Any mainstream version | -                                                         |
+| libstdc++-static | Any mainstream version | Only needed in CentOS 8+, RedHat 8+, and Fedora systems.  |
+| libasan          | Any mainstream version | Only needed in CentOS 8+, RedHat 8+, and Fedora systems.  |
+
+## Requirements and suggestions for installing Nebula Graph in test environments
+
+### Hardware requirements for test environments
+
+| Item               | Requirement |
+| ------------------ | ----------- |
+| CPU architecture   | x86_64      |
+| Number of CPU core | 4           |
+| Memory             | 8 GB        |
+| Storage            | 100 GB, SSD |
+
+### Supported operating systems for test environments
+
+For now, we can only install Nebula Graph in the Linux system. To install Nebula Graph in a test environment, we recommend that you use any Linux system with kernel version 2.6.32 or above.
+
+You can adjust some of the kernel parameters to better accommodate the need for running Nebula Graph. For more information, see [Optimize Linux kernel configuration](TODO) [TODO].
+
+### Suggested service architecture for test environments
+
+| Process                                   | Suggested number |
+| ----------------------------------------- | ---------------- |
+| metad (the metadata service process)      | 1                |
+| storaged (the storage service process)    | 1 or more        |
+| graphd (the query engine service process) | 1 or more        |
+
```

Comment:
For example, for a single machine env, you can deploy as follows:
 A 1 1 1

For a more common example, for a 3 machine, ....